hub/saml-proxy/src/main/java/uk/gov/ida/hub/samlproxy/SamlProxyApplication.java

```diff
@@ -51,21 +57,39 @@ public final void initialize(Bootstrap<SamlProxyConfiguration> bootstrap) {
                 )
         );
 
+        bootstrap.addBundle(verifyMetadataBundle);
         guiceBundle = defaultBuilder(SamlProxyConfiguration.class)
-                .modules(new SamlProxyModule(), new EventEmitterModule())
+                .modules(new SamlProxyModule(), new EventEmitterModule(), bindVerifyMetadata())
                 .build();
         bootstrap.addBundle(guiceBundle);
         bootstrap.addBundle(new ServiceStatusBundle());
         bootstrap.addBundle(new MonitoringBundle());
         bootstrap.addBundle(new LoggingBundle());
     }
 
+    private AbstractModule bindVerifyMetadata() {
```

Comment:
It would be nice if this Module was provided by one of our libs and didn't depend on constants in `SamlProxyModule`.